docs/rep/controller/openapi.yaml

```diff
@@ -0,0 +1,370 @@
+openapi: 3.0.2
+info:
+  title: Rik
+  version: 0.1.0
+paths:
+  /api/v0/workloads.list:
+    get:
+      tags:
+        - Workloads
+      description: List all workloads
+      parameters:
+        - required: false
+          schema:
+            title: limit
+            minimum: 1
+            type: integer
+          name: limit
+          in: query
+      responses:
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                type: object
+                properties:
+                  workloads:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Workload'
+  /api/v0/workloads.create:
+    post:
+      tags:
+        - Workloads
+      description: Create a new workload
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/WorkloadDefinition'
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            text/plain:
+              schema:
+                $ref: '#/components/schemas/WorkloadName'
```

Comment:
Is the workload name passed through the `WorkloadDefinition` mandatory? If so, returning one through the response is redundant.